mysql-test/suite/tokudb.rpl/t/rpl_xa_interleave.test

```diff
@@ -0,0 +1,104 @@
+--source include/have_tokudb.inc
+--source include/not_group_replication_plugin.inc
+--source include/have_binlog_format_mixed_or_row.inc
+--source include/master-slave.inc
```

Comment:
include/count_sessions.inc ?

Reply:
Never-mind, my eyes were crossed and seeing this as session not rpl connections...